summaries/Evolutionary-testing-of-classes-summary.md

```diff
@@ -0,0 +1,81 @@
+# Evolutionary Testing of Classes
+**Article:**
+*Tonella, Paolo*. Evolutionary testing of classes. In Proceedings of the 2004 ACM SIGSOFT international symposium on Software testing and analysis (ISSTA '04). ACM, New York, NY, USA, 119-128.
+
+
+## Automated Test Case Generation
+Consists of creating an object, changing the state of that object and method call with proper input values. 
+Unit tests of classes should include all possible constructor and method invocation in order all cases to be analyzed.
+Generic algorithms use chromosome encoding ensuring that for hard to reach code fragments test cases are implemented by creating sequence of program calls.
```

Comment:
Maybe simplify the structure of the sentence a bit by leaving some details out:
`Genetic algorithms use chromosome encoding to unsure test coverage of hard-to-reach code fragments.`